www/Apptentive.js

```diff
@@ -1,107 +1,245 @@
+      "ApptentiveBridge",
+      "getPersonEmail",
+      []
+    );
+  },
+
+  setPersonEmail: function (successCallback, errorCallback, email) {
+    cordova.exec(
+      successCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "setPersonEmail",
+      [email]
+    );
+  },
+
+  getPersonName: function (successCallback, errorCallback) {
+    cordova.exec(
+      successCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "getPersonName",
+      []
+    );
+  },
+
+  setPersonName: function (successCallback, errorCallback, name) {
+    cordova.exec(
+      successCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "setPersonName",
+      [name]
+    );
+  },
+
+  setRatingProvider: function (
+    successCallback,
+    errorCallback,
+    ratingProviderName
+  ) {
+    successCallback(); // Deprecated. Use the ANDROID_CUSTOM_APP_STORE_URL variable instead.
+  },
+
+  addUnreadMessagesListener: function (unreadMessagesCallback, errorCallback) {
+    cordova.exec(
+      unreadMessagesCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "addUnreadMessagesListener",
+      []
+    );
+  },
+
+  addSurveyFinishedListener: function (surveyFinishedCallback, errorCallback) {
+    cordova.exec(
+      surveyFinishedCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "setOnSurveyFinishedListener",
+      []
+    );
+  },
+
+  showMessageCenter: function (successCallback, errorCallback, customData) {
+    if (customData) {
+      cordova.exec(
+        successCallback,
+        errorCallback,
+        "ApptentiveBridge",
+        "showMessageCenter",
+        [customData]
+      );
+    } else {
+      cordova.exec(
+        successCallback,
+        errorCallback,
+        "ApptentiveBridge",
+        "showMessageCenter",
+        []
+      );
+    }
+  },
+
+  canShowMessageCenter: function (successCallback, errorCallback) {
+    cordova.exec(
+      successCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "canShowMessageCenter",
+      []
+    );
+  },
+
+  canShowInteraction: function (successCallback, errorCallback, eventName) {
+    cordova.exec(
+      successCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "canShowInteraction",
+      [eventName]
+    );
+  },
+
+  setProperty: function (successCallback, errorCallback, key, value) {
+    successCallback(); // Does nothing on Android
+  },
+
+  login: function (successCallback, errorCallback, token) {
+    successCallback(); // Does nothing on Android. We are working on re-implementing this feature.
+  },
+
+  logout: function (successCallback, errorCallback) {
+    successCallback(); // Does nothing on Android. We are working on re-implementing this feature.
+  },
+
+  sendAttachmentText: function (successCallback, errorCallback, text) {
+    cordova.exec(
+      successCallback,
+      errorCallback,
+      "ApptentiveBridge",
+      "sendAttachmentText",
+      [text]
+    );
+  },
+
```

Comment:
This is the only function I added here. The rest is my editor decided to reformat, I am not able to get rid of those changes.